lib/zpool.js

```diff
@@ -0,0 +1,204 @@
+var cp = require('child_process');
+var util = require('./util');
+
+
+
+function ZPool(info) {
+    "use strict";
+
+    var obj = Object.create({});
+    if (typeof info === 'string') {
+        info = info.split(/\s+/);
+    }
+
+    if (info.length !== 10) {
+        return null;
+    }
+
+    obj.name = info[0];
+    obj.size = info[1];
+    obj.alloc = info[2];
+    obj.free = info[3];
+    obj.cap = info[4];
+    obj.health = info[5];
+    if (info[6] !== '-') {
+        obj.altroot = info[6];
+    }
+
+    Object.freeze(obj);
+    return obj;
+}
+
+function analyzeVdevOpts (deviceList) {
```

Comment:
This should probably be renamed, if it deserves to exist on its own? It does not actually analyze any vdev opts. :)